packages/utils/src/RBAC.js

```diff
@@ -0,0 +1,93 @@
+
+export async function getRBAC(applicationName = '') {
+    const insights = window.insights;
+    if (
+        insights &&
+    insights.chrome &&
+    insights.chrome.auth &&
+    insights.chrome.auth.getUser &&
+    insights.chrome.getUserPermissions
+    ) {
+        const user = await insights.chrome.auth.getUser();
+        const permissions = await insights.chrome.getUserPermissions(applicationName);
+        return {
+            isOrgAdmin: user.identity.user.is_org_admin,
+            permissions
+        };
+    }
+
+    return { isOrgAdmin: false, permissions: null };
+}
+
+export function doesHavePermissions(RBACResults, permissionList) {
```

Comment:
I wouldn't mix orgAdmin and permission together when checking for permission access. User can have access to some parts if he's not org admin but has permissions nad vie versa.

```JS
export function doesHavePermissions(userPermissions, permissionList) {
  return userPermissions.some((access) => permissionList.includes(access.permission));
}
```